dotcom-rendering/src/components/MultiByline.tsx

```diff
@@ -0,0 +1,248 @@
+	format,
+	fontWeight,
+}: {
+	format: ArticleFormat;
+	fontWeight: 'light' | 'medium';
+}) => css`
+	${format.display === ArticleDisplay.Immersive
+		? fontWeight === 'light'
+			? headlineLightItalic28
+			: headlineMediumItalic28
+		: fontWeight === 'light'
+		? headlineLightItalic24
+		: headlineMediumItalic24};
+
+	${from.tablet} {
+		${format.display === ArticleDisplay.Immersive
+			? fontWeight === 'light'
+				? headlineLightItalic34
+				: headlineMediumItalic34
+			: fontWeight === 'light'
+			? headlineLightItalic28
+			: headlineMediumItalic28}
+	}
+
+	/** Labs uses sans text */
+	${format.theme === ArticleSpecial.Labs &&
+	css`
+		${format.display === ArticleDisplay.Immersive
+			? textSansItalic28
+			: textSansItalic24}
+
+		${from.tablet} {
+			${format.display === ArticleDisplay.Immersive
+				? textSansItalic34
+				: textSansItalic28}
+		}
+	`}
+`;
+
+const nonAnchorHeadlineStyles = (format: ArticleFormat) => {
+	switch (format.design) {
+		case ArticleDesign.Obituary:
+		case ArticleDesign.Comment:
+		case ArticleDesign.Editorial:
+			return fontStyles({ format, fontWeight: 'light' });
+
+		case ArticleDesign.Standard:
+		case ArticleDesign.Profile:
+		case ArticleDesign.Explainer:
+		case ArticleDesign.Timeline:
+		case ArticleDesign.LiveBlog:
+		case ArticleDesign.DeadBlog:
+		case ArticleDesign.Analysis:
+		case ArticleDesign.Feature:
+		case ArticleDesign.Interview:
+		case ArticleDesign.Recipe:
+		case ArticleDesign.Review:
+			return fontStyles({ format, fontWeight: 'medium' });
+
+		default:
+			return fontStyles({ format, fontWeight: 'medium' });
+	}
+};
+
+const bylineStyles = (format: ArticleFormat) => css`
+	${nonAnchorHeadlineStyles(format)}
+	margin-top: -4px;
+	color: ${palette('--multi-byline-non-linked-text')};
+	a {
+		${subheadingStyles(format)}
+		color: ${palette('--link-kicker-text')};
+		text-decoration: none;
+		font-style: normal;
+		:hover {
+			text-decoration: underline;
+		}
+	}
+	span[data-contributor-rel='author'] {
+		${subheadingStyles(format)}
+		color: ${palette('--multi-byline-non-linked-text')};
+		font-style: normal;
+	}
+`;
+
+const bylineWrapperStyles = css`
+	display: flex;
+	width: 100%;
+	overflow: hidden;
+	border-bottom: 1px solid ${palette('--article-border')};
+	margin-bottom: ${space[2]}px;
+`;
+
+const bylineTextStyles = css`
+	flex-grow: 1;
+	margin-bottom: ${space[6]}px;
+	${from.tablet} {
+		margin-bottom: ${space[9]}px;
+	}
+`;
+
+const bylineImageStyles = css`
+	width: 80px;
+	border-radius: 50%;
+	margin-left: 10px;
+	margin-bottom: -8px;
+	height: 80px;
+	min-width: 80px;
+	overflow: hidden;
+	align-self: flex-end;
+	${from.tablet} {
+		height: 120px;
+		min-width: 120px;
+		width: 120px;
+		margin-bottom: -12px;
+	}
+`;
+
+interface MultiBylineItemProps {
+	multiByline: MultiBylineModel;
+	format: ArticleFormat;
+	tags: TagType[];
+	children: React.ReactNode;
+}
+
+export const MultiByline = ({
+	multiByline,
+	format,
+	tags,
+	children,
+}: MultiBylineItemProps) => {
```

Comment:
I think `MultiBylineItem` was an earlier name for this component, but now isn't used? In fact you can probably just call this `Props` without qualification since this is the "main" component exported from this file?

```suggestion
interface Props {
	multiByline: MultiBylineModel;
	format: ArticleFormat;
	tags: TagType[];
	children: React.ReactNode;
}

export const MultiByline = ({
	multiByline,
	format,
	tags,
	children,
}: Props) => {
```